src/packages/@ncigdc/modern_components/ClinicalAnalysis/ClinicalVariableCard/ContinuousVariableCard.js

```diff
@@ -496,19 +496,27 @@ export default compose(
       props.variable.isSurvivalCustom === nextProps.variable.isSurvivalCustom
     ),
     ({
-      defaultData: { bins },
+      binsAreCustom,
+      defaultData: { bins: defaultBins },
       dispatch,
       fieldName,
       id,
+      variable: { bins },
     }) => ({
       resetBins: () => {
         dispatch(updateClinicalAnalysisVariable({
           fieldName,
           id,
           variable: {
-            bins,
             ...resetVariableDefaults.survival,
-            ...resetVariableDefaults.continuous,
+            ...binsAreCustom
+              ? {
+                bins: defaultBins,
+                ...resetVariableDefaults.continuous,
+              }
+              : {
```

Comment:
this could be an empty object no? don't have to update `variable.bins` if the bins aren't custom

Reply:
if you do that, it returns the overall data.

Reply:
on second thought 🤔 let me go try that... I'm not so sure you're not 100% right. brb